ucm2/Qualcomm/x1e80100/x1e80100.conf

```diff
@@ -1,12 +1,12 @@
 Syntax 4
 
-If.LENOVOX14s {
+If.LENOVOT14s {
 	Condition {
 		Type RegexMatch
 		String "${sys:devices/virtual/dmi/id/board_vendor}-${sys:devices/virtual/dmi/id/product_family}"
 		Regex "LENOVO.*ThinkPad T14s Gen 6.*"
```

Comment:
Change to `Regex "(LENOVO.*ThinkPad T14s Gen 6.*)|(HP.*Omnibook X.*)"` and remove new If block.

Reply:
Well I tried just that the last time, and it didn't work. Will try again.

Reply:
And this time it worked :grin: Thank you. I had to redo the whole box first, because... reasons